source/linux/Foundational_Components/Kernel/Kernel_Drivers/QSPI.rst

```diff
@@ -273,102 +245,106 @@ refer to :file:`Documentation/devicetree/bindings/mtd/jedec,spi-nor.yaml`,
 :file:`Documentation/devicetree/bindings/mtd/spi-nand.yaml` and
 :file:`Documentation/devicetree/bindings/mtd/partitions/partition.yaml`.
 
-Following is an example device tree describing OSPI NOR
-
-.. code-block:: dts
-
-   &ospi0 {
-
-      flash@0{
-         compatible = "jedec,spi-nor";
-         reg = <0x0>;
-         spi-tx-bus-width = <8>;
-         spi-rx-bus-width = <8>;
-         spi-max-frequency = <25000000>;
-         cdns,tshsl-ns = <60>;
-         cdns,tsd2d-ns = <60>;
-         cdns,tchsh-ns = <60>;
-         cdns,tslch-ns = <60>;
-         cdns,read-delay = <4>;
-
-         partitions {
-            compatible = "fixed-partitions";
-            #address-cells = <1>;
-            #size-cells = <1>;
-            bootph-all;
-
-            partition@0 {
-               label = "ospi.tiboot3";
-               reg = <0x00 0x80000>;
-            };
+.. ifconfig:: CONFIG_part_variant in ('AM64X', 'AM62X', 'AM62PX', 'J7200', 'J721E', 'J722S', 'J721S2', 'J784S4','J742S2')
 
-            partition@80000 {
-               label = "ospi.tispl";
-               reg = <0x80000 0x200000>;
-            };
+   The following is an example device-tree node for an OSPI NOR device
+
+   .. code-block:: dts
```

Comment:
`dts` is not a recognized literal block. I get this warning:

`/home/suhaas/git/processor-sdk-doc/source/linux/Foundational_Components/Kernel/Kernel_Drivers/QSPI.rst:278: WARNING: Could not lex literal_block as "dts". Highlighting skipped.`
Instead you could keep it just as code-block.

Reply:
Oh no, it is a recognized block. That error just means it couldn't be parsed as written. Incomplete syntax.

https://pygments.org/docs/lexers/#pygments.lexers.devicetree.DevicetreeLexer